studygroups/views/api.py

```diff
@@ -229,6 +231,15 @@ def get(self, request):
                 default=F('first_meeting_date_value'),
                 output_field=CharField(),
             )
+        ).annotate(
+            status=Case(
+                When(signup_open=True, first_meeting_date__gt=today, then=Value('upcoming')),
+                When(signup_open=True, first_meeting_date__lte=today, last_meeting_date__gte=today, then=Value('in_progress')),
+                When(signup_open=False, then=Value('closed')),
```

Comment:
Wouldn't this label any completed learning circle with `signup_open == False` as `"closed"` rather than completed?